spec/unit/remote_system_spec.rb

```diff
@@ -68,12 +73,28 @@
 
         remote_system.run_command("ls", :disable_logging => true)
       end
+
+      it "adheres to the remote_user option" do
+        expect(Cheetah).to receive(:run).with("ssh", "machinery@remotehost", "sudo", "LC_ALL=C", "ls", "/tmp", {})
+
+        remote_system.remote_user = "machinery"
+        remote_system.run_command("ls", "/tmp")
+      end
     end
 
     describe "#retrieve_files" do
       it "retrieves files via rsync from a remote host" do
-        expect(Cheetah).to receive(:run).with("rsync", "-e", "ssh", "--chmod=go-rwx", "--files-from=-", "root@remotehost:/", "/tmp",  :stdout => :capture, :stdin => "/foo\n/bar" )
+        expect(Cheetah).to receive(:run).with("rsync", "-e", "ssh", "--chmod=go-rwx", "--files-from=-", "--rsync-path=rsync", "root@remotehost:/", "/tmp",  :stdout => :capture, :stdin => "/foo\n/bar" )
```

Comment:
Line is too long. [201/100]<br>Unnecessary spacing detected.<br>Use the new Ruby 1.9 hash syntax.<br>Space inside parentheses detected.